kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(user_agent):
+    """Get Browser Name from User Agent"""
+
+    user_agent = str(user_agent)
+
+    match = re.search(r'(?i)(firefox|msie|chrome|safari|trident)[\/\s]([\d.]+)', user_agent)
```

Comment:
1. Oh! Sorry, I forgot to remove it. It was used to get the version of the browser. As we dont need the version anymore, I think its ok to remove it.

2. Thanks for this. I did not have idea about this. I have corrected it in next commit.